components/brave_wallet_ui/panel/reducers/panel_reducer.ts

```diff
@@ -87,36 +82,14 @@ export const createPanelReducer = (initialState: PanelState) => {
   reducer.on(
     PanelActions.navigateTo.type,
     (state: PanelState, selectedPanel: PanelTypes) => {
-      const foundTitle = PANEL_TITLES.find(
-        (title) => selectedPanel === title.id
-      )
-      const panelTitle = foundTitle ? foundTitle.title : ''
       return {
         ...state,
         selectedPanel,
-        lastSelectedPanel: state.selectedPanel,
-        panelTitle
+        lastSelectedPanel: state.selectedPanel
```

Comment:
It seems lastSelectedPanel is never read now

Reply:
Removed!